util/database.ts

```diff
@@ -0,0 +1,16 @@
+import { DataSource } from "typeorm";
+import { Movie } from "../model/Movie";
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export const AppSource = new DataSource({
+  type: "postgres",
+  host: "localhost",
+  port: 5432,
+  username: process.env.DATABASE_USERNAME,
```

Comment:
Create config.ts file where you will read all  process.env, check if they have value and export them in a better format. Like config object containing all .env files organized by group etc...
export const config = {db: {// allDbConfig files here}, goTinyUrl: string}

Reply:
Fixed.